src/main/java/de/plushnikov/intellij/plugin/handler/BuilderHandler.java

```diff
@@ -0,0 +1,21 @@
+package de.plushnikov.intellij.plugin.handler;
+
+import com.intellij.codeInsight.daemon.impl.HighlightInfo;
+import com.intellij.psi.PsiField;
+import com.intellij.psi.PsiFile;
+import com.intellij.psi.util.PsiTreeUtil;
+import lombok.Builder;
+import de.plushnikov.intellij.plugin.util.PsiAnnotationSearchUtil;
+
+
+public class BuilderHandler {
+
+  public static boolean isDefaultBuilderValue(HighlightInfo highlightInfo, PsiFile file) {
```

Comment:
You can add here @NotNull annotations too.

Reply:
Yes, sure